components/Counter.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export default function Counter({ max, counter }) {
+  return (
+    <>
+      {counter < max && counter >= 0 ? (
+        <CharacterCounter> {max - counter} Zeichen</CharacterCounter>
+      ) : (
+        <CharacterCounter>
+          Es sind keine Zeichen mehr übrig - bitte kürze die Beschreibung
+        </CharacterCounter>
+      )}
+    </>
+  );
```

Comment:
If you want to destinguish between these two cases, its best to move the ternary opterator to the exact part which is dynamic, in this case the text inside the CharacterCounter:
```js
  return (
      <CharacterCounter> 
        {counter < max && counter >= 0 
            ? `${max - counter} Zeichen übrig`
            : "Es sind keine Zeichen mehr übrig - bitte kürze die Beschreibung"
        }
      </CharacterCounter>
  );
```

Reply:
changed that😃